app/views/content_items/detailed_guide.html.erb

```diff
@@ -7,9 +7,7 @@
     <%= render 'govuk_component/title', @content_item.title_and_context %>
   </div>
   <div class="column-third">
-    <%= render 'shared/available_languages',
-      translations: @content_item.available_translations
-    %>
+    <%= render 'components/translation-nav', translations: @content_item.available_translations %>
```

Comment:
Take a look at how government-frontend handles the title, metadata and document_footer components.